src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -190,7 +225,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
         const channelType = channel?.channelType === 'GROUP' ? 'group' : 'messaging';
         response = await SignedMessageRepo.getSignedChannelDetail(channelType, channel?.id);
       }
-      await helperSaveChannelDetail(channel, response);
+      helperSaveChannelDetail(channel, response);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `helperSaveChannelDetail` function is now being called without awaiting its result. This could potentially lead to race conditions if the results of this function are needed before proceeding with the next steps. If the function needs to be awaited, consider adding back the `await` keyword.



<!-- This is an auto-generated comment by OSS CodeRabbit -->